conf/drupal.go

```diff
@@ -20,14 +23,29 @@ func GetRootPath() string {
 		return filepath.Join(wd, c)
 	}
 
-	rootPath, version := scanForCoreDrupal()
+	log.Fatalf("Drupal path setting is missing.")
+	return ""
+
+	// rootPath, version := scanForCoreDrupal()
+
+	// CreateOrReplaceDrupalConfig(strings.Replace(rootPath, wd, "", -1), version)
+
+	// return rootPath
```

Comment:
Should these be commented? All good to remove if they’re no longer used, but not sure how it works if it only ever returns an empty string